pkg/certs/cert-inspection/certgraphanalysis/nameguessing.go

```diff
@@ -7,56 +7,7 @@ import (
 	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
 )
 
-// TODO these should all be eliminated in favor of self-describing annotations.
-
-func guessLogicalNamesForCertKeyPairList(in *certgraphapi.CertKeyPairList, nodes map[string]int) {
-	for i := range in.Items {
-		meaning := guessMeaningForCertKeyPair(in.Items[i], nodes)
-		in.Items[i].LogicalName = meaning.name
-		in.Items[i].Description = meaning.description
-	}
-}
-
-func newSecretLocation(namespace, name string) certgraphapi.InClusterSecretLocation {
-	return certgraphapi.InClusterSecretLocation{
-		Namespace: namespace,
-		Name:      name,
-	}
-}
-
-var secretLocationToLogicalName = map[certgraphapi.InClusterSecretLocation]logicalMeaning{
```

Comment:
Are these logically per-location or per content?  Where are you moving them to?

Reply:
Per-location. Description is now in origin's `tls/rawTLSInfo`

Reply:
> Per-location. Description is now in origin's `tls/rawTLSInfo`

We need some kind of historical to find and fix all the descriptions.  You want to create an issue or open PRs for each of these?